File: ws/loza/u1/New Folder/ChickenFarmSimuladorx/src/ec/edu/espe/chickenfarmsimulador/model/Poop.java
package ec.edu.espe.chickenfarmsimulador.model;

/**
 *
 * @author Joseph Medina
 */
public class Poop {
    private int amount;
   
    @Override
    public String toString() {
        return "Poop {" + "amount = " + amount + '}';
    }
   
    public Poop(int amount) {
        this.amount = amount;
    }

    /**
     * @return the amount
     */
    public int getAmount() {
        return amount;
    }

    /**
     * @param amount the amount to set
     */
    public void setAmount(int amount) {
        this.amount = amount;
    }
   
   
}    

File: hw/medina/u1/HW11ChickenCoops/newChickenCoopJS/model/Poop.js
/**
 * Poop class
 * Represents the poop from a chicken.
 */
class Poop {
  /**
   * @param {number} amount
   */
  constructor(amount) {
    this.amount = amount;
  }

  toString() {
    return `Poop{ amount = ${this.amount} }`;
  }
}

module.exports = { Poop };
